symphony-bdk-core/src/main/java/com/symphony/bdk/core/SymphonyBdk.java

```diff
@@ -142,12 +149,31 @@ public AuthSession obo(String username) throws AuthUnauthorizedException {
   }
 
   /**
-   * Extension App Authenticate by app token
-   * @param appToken App Token
-   * @return Extension App authentication session
+   * Returns the {@link ExtensionAppAuthenticator}.
+   *
+   * @return the {@link ExtensionAppAuthenticator}
+   */
+  public ExtensionAppAuthenticator appAuthenticator() {
```

Comment:
why you want to expose the AppAuthenticator from SymphonyBdk?

Reply:
That's not perfect I agree, but before that the `SymphonyBdk` was not providing a way to implement the Circle Of Trust. We might have to revamp a bit this feature